pkg/tfbridge/provider.go

```diff
@@ -379,7 +380,7 @@ func (p *Provider) Configure(ctx context.Context,
 	var missingKeys []*pulumirpc.ConfigureErrorMissingKeys_MissingKey
 	for key, meta := range p.config {
 		if meta.Required && !config.IsSet(key) {
-			name := TerraformToPulumiName(key, meta, false)
+			name := TerraformToPulumiName(key, meta, nil, false)
```

Comment:
We do have schemainfo for config, right?  Is it not used here because we don’t (yet?) support maxitemsone annotations there?

Reply:
In config annotations, we *only* support primitive types as of now